spec/controllers/idv/inherited_proofing_cancellations_controller_spec.rb

```diff
@@ -106,31 +106,54 @@
     end
 
     context 'when there is a session' do
+      subject(:action) do
+        get :new, params: { step: step }
+      end
+
       before do
         stub_sign_in
       end
 
       it 'renders template' do
-        get :new, params: { step: step }
+        subject
```

Comment:
I believe the subject var name (`action`) is preferred, if not in newer versions of rubocop I believe.